stacklight_tests/clients/influxdb_grafana_api.py

```diff
@@ -23,6 +45,26 @@ def __init__(self, address, port, username, password, db_name):
 
         self.influx_db_url = "http://{0}:{1}/".format(self.address, self.port)
 
+    @staticmethod
+    def compile_query(query, replaces):
+        for pattern, value in replaces.items():
+            query = query.replace(pattern, value)
+        # NOTE(rpromyshlennikov): temporary fix for unknown hostname
+        # (node-1 vs node-1.test.domain.local)
+        query = query.replace(".test.domain.local", "")
+        # NOTE(rpromyshlennikov): fix for regex queries (e.g: for mount points)
+        query = query.replace("^/", "^\/")
+        return query
+
+    @staticmethod
+    def parse_measurement(query):
+        result = re.search('from \"*(\w+)\"*', query, re.IGNORECASE)
```

Comment:
It is really strange, that I have put here "*", because I just moved this code from one class to another and don't remember, that I change it. Sorry, done.